### PR TITLE
Support export csv in HPA-UMAP-3D

### PR DESCRIPTION
**Pull request by cwinsnes**
Add simple string csv export for filtered data

### CODE REVIEW

File: src/HPA-UMAP-3D.imjoy.html [hunk 1]
```diff
@@ -132,7 +132,20 @@
...
+                        export_string += current_data["x"] + ","
+                        export_string += current_data["y"] + ","
+                        export_string += current_data["z"] + ","
+                        export_string += current_data["gene"] + ","
+                        export_string += "\"" + current_data["location"] + "\","
+                        export_string += current_data["x"] + "," 
+                        export_string += 'https://www.proteinatlas.org/'+current_data["gene_ensembl_id"]+'-'+current_data["gene"]+'/cell/'
+                        export_string += "\n"
+                    }
+                    api.exportFile(export_string, 'export.csv')
```
Comment (oeway) [inline src/HPA-UMAP-3D.imjoy.html]: As we discussed, perhaps better to use ; as separate for csv file.

**Review by oeway — CHANGES_REQUESTED** [review]
Please also bump the version number before it get merged.

File: src/HPA-UMAP-3D.imjoy.html (hunk 1 above)
Comment (cwinsnes) [reply src/HPA-UMAP-3D.imjoy.html]: That is untrue.  With the filename csv you should definitely use commas. It is easy to use in Excel too, despite the commas and Microsofts aversion to conforming to any kind of standards.

**Review by oeway — APPROVED** [review]

File: src/HPA-UMAP-3D.imjoy.html (hunk 1 above)
Comment (oeway) [reply src/HPA-UMAP-3D.imjoy.html]: Here is a thread about it: https://stackoverflow.com/questions/10140999/csv-with-comma-or-semicolon

With also one solution for MS Excel: https://stackoverflow.com/questions/17953679/how-to-correctly-display-csv-files-within-excel-2013#19769403
Not sure how that behaves on other reader though.

**Comment by oeway** [comment]
BTW, why not use the original header and export all fields?

**Comment by cwinsnes** [comment]
These fields are what the people who asked for the export wanted. You are right that it probably makes sense to add all of them but I'd rather do that a bit later as I should not spend too much time on this today.

File: src/HPA-UMAP-3D.imjoy.html (hunk 1 above)
Comment (cwinsnes) [reply src/HPA-UMAP-3D.imjoy.html]: I would rather follow the RFC (https://www.ietf.org/rfc/rfc4180.txt) than follow Microsofts bullshit.

Every other reader I have tried immediately recognizes commas, like they should.

**Pull request merged**